motoman_driver/src/io_relay.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ * Copyright (c) 2013, Southwest Research Institute
```

Comment:
Update copyright here as well.

And as this includes code moved from the original location, it'd be nice to add Delft Universities copyright as well

Reply:
Addressed in 8c1757d